package.json

```diff
@@ -108,7 +108,7 @@
     "use-cjs": "echo '{\n  \"extends\": \"./tsconfig/cjs.json\"\n}' > tsconfig.json",
     "use-mjs": "echo '{\n  \"extends\": \"./tsconfig/mjs.json\"\n}' > tsconfig.json",
     "=============================================================================== aliases": "",
-    "test": "cd jest && pnpm test",
+    "test": "cd testsuite && pnpm test",
```

Comment:
I was going to make a PR for this after you merged the test suite, but you beat me to it.

I would ask, however, that you make this

``` suggestion
    "test": "cd testsuite && pnpm test --",
```

instead.  The reason for this is so that you can pass options on to the test script in the testsuite directory.  For example, in that directory, you can do

``` shell
pnpm -s test -- --collectCoverageFrom 'mjs/input/tex/ams/*.js' tests/input/tex/Ams.test.js
```

in order to restrict the coverage listing to be only the files in the ams package.  With the addition of the `--` in the package script, you can do the same from the top-level directory.  Without it, you would have to use

``` shell
pnpm -s test -- -- --collectCoverageFrom 'mjs/input/tex/ams/*.js' testsuite/tests/input/tex/Ams.test.js
```

with two `--`.  Adding one into the script allows the same format to be used in both directories.

Reply:
Actually

``` suggestion
    "test": "cd testsuite && pnpm -s test --"
```

is even better.